changes/metadata_test.go

```diff
@@ -0,0 +1,391 @@
+package changes
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
+	// testdata already has .changes-static, but make another .changes directory so that we can easily cleanup after
+	// tests have run.
+	err := os.MkdirAll("testdata/.changes/next-release", 0777)
```

Comment:
Since this will be dynamic data, should we just create a temporary directory using `ioutil.TempDir`? `testdata` is good for populating static data to be used for tests, but if we are going to be manipulating the directory contents then we should use a temporary directory in a different location that we can cleanup. This will prevent accidental changes to the static testdata that might get committed int the repository if cleanup fails.